vagrant/ansible/Makefile

```diff
@@ -14,7 +14,7 @@ setup.clients:
 	@ansible-playbook -i $(INVENTORY) ${ANSIBLE_EXTRA_VARS} ./setup-clients.yml
 
 client.test:
-	@ansible-playbook -i $(INVENTORY) ${ANSIBLE_EXTRA_VARS} ./client-test.yml
+	@ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook -i $(INVENTORY) ${ANSIBLE_EXTRA_VARS} ./client-test.yml
```

Comment:
Isn't this one sufficient?

Reply:
you are right. Will fix and repush new changes.